### ISSUE TITLE
"Swap Start (aka Command) and Alt keys on Windows keyboard" doesn't work correctly.

### ISSUE THREAD

**Issue by ethaniel** [tekezo/Karabiner#840]
I would expect this option:

![image](https://user-images.githubusercontent.com/941973/27378586-9608753c-56a2-11e7-875d-147ecaada495.png)

to work only when a USB keyboard is connected. But turns out it also alters the behaviour on the built-in macbook keyboard as well (swapping the option and command keys).

So, in my scenario, it perfectly swaps the external keyboard keys when I'm at home. But when I'm on the road, I have to manually disable this function, because my built-in macbook keys are switched as well.

**Comment by dunkarooftop** [comment]
You can write your own rule to limited it only effect your external keyboard.

https://pqrs.org/osx/karabiner/xml.html.en
search "Filter: Specify device"

**Comment by ethaniel** [comment]
True. But if Karabiner says that it's a "IBM model M keyboard" then it's definately an external one. So this option is configured incorrectly.
